db_scripts/dump.sh

```diff
@@ -1,6 +1,31 @@
 #!/bin/bash
 
-# For AMI: sudo yum -y install https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6-x86_64/pgdg-ami201503-96-9.6-2.noarch.rpm
-#          sudo yum -y install postgresql96
+set -eu -o pipefail
+
+ENV_NAME=prod
```

Comment:
How would the `if` check below ever be false with this assignment?

Reply:
It wouldn't. In other contexts, ENV_NAME is allowed to be dev, test, stage or prod (such as in the restore script, and in the env vars for the fargate task definition). I'm using it here to form the secret ID and get the DB password out of secretsmanager. I'm not sure, other than hard-coding the RDS endpoint, how to confirm that we're dumping the prod database.